modules/memfault-firmware-sdk/memfault_integration_coredump.c

```diff
@@ -0,0 +1,35 @@
+#include <zephyr/kernel.h>
+#include <memfault/components.h>
+#include <memfault/ports/zephyr/coredump.h>
+#include "memfault_etb_trace_capture.h"
+
+static sMfltCoredumpRegion s_coredump_regions[MEMFAULT_ZEPHYR_COREDUMP_REGIONS + IS_ENABLED(CONFIG_MEMFAULT_NCS_ETB_CAPTURE)];
+
+const sMfltCoredumpRegion
+*memfault_platform_coredump_get_regions(const sCoredumpCrashInfo *crash_info, size_t *num_regions)
```

Comment:
This is just to provide an example?

Reply:
Oh whoops, yeah need to figure out what to do here because I don't think using a weak attribute will work.